tests/integration/base_deploy_test.go

```diff
@@ -34,30 +39,53 @@ var (
 )
 
 //Test to make sure all rook components are installed and Running
-func checkIfRookClusterIsInstalled(k8sh *utils.K8sHelper, s suite.Suite, oNamespace string, cNamespace string) {
-	logger.Infof("Make sure all Pods in Rook Cluster %s are running", cNamespace)
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-operator", oNamespace, 1, "Running"),
+func checkIfRookClusterIsInstalled(s suite.Suite, k8sh *utils.K8sHelper, opNamespace string, clusterNamespace string) {
+	logger.Infof("Make sure all Pods in Rook Cluster %s are running", clusterNamespace)
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-operator", opNamespace, 1, "Running"),
 		"Make sure there is 1 rook-operator present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-agent", oNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-agent", opNamespace, 1, "Running"),
 		"Make sure there is 1 rook-agent present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-api", cNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-api", clusterNamespace, 1, "Running"),
 		"Make sure there is 1 rook-api present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mgr", cNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mgr", clusterNamespace, 1, "Running"),
 		"Make sure there is 1 rook-ceph-mgr present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-osd", cNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-osd", clusterNamespace, 1, "Running"),
 		"Make sure there is at lest 1 rook-ceph-osd present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mon", cNamespace, 3, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mon", clusterNamespace, 3, "Running"),
 		"Make sure there are 3 rook-ceph-mon present in Running state")
 }
 
-func gatherAllRookLogs(k8sh *utils.K8sHelper, s suite.Suite, oNamespace string, cNamespace string) {
-	logger.Infof("Gathering all logs from Rook Cluster %s", cNamespace)
-	k8sh.GetRookLogs("rook-operator", oNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-agent", oNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-api", cNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-ceph-mgr", cNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-ceph-mon", cNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-ceph-osd", cNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-ceph-rgw", cNamespace, s.T().Name())
-	k8sh.GetRookLogs("rook-ceph-mds", cNamespace, s.T().Name())
+func checkIfRookClusterIsHealthy(s suite.Suite, testClient *clients.TestClient, clusterNamespace string) {
+	logger.Infof("Testing cluster %s health", clusterNamespace)
+	var err error
+	var status model.StatusDetails
+
+	retryCount := 0
+	maxRetries := 25
```

Comment:
there are constants used elsewhere in the tests for retry and sleep time

Reply:
OK i will use the consts, i'm not sure why i didn't.